firebase.js

```diff
@@ -0,0 +1,10 @@
+const firebaseConfig = {
+	apiKey: "YOUR_API_KEY",
+	authDomain: "YOUR_AUTH_DOMAIN",
+	projectId: "YOUR_PROJECT_ID",
+	storageBucket: "YOUR_STORAGE_BUCKET",
+	messagingSenderId: "YOUR_MESSAGING_SENDER_ID",
+	appId: "YOUR_APP_ID",
+};
+
+firebase.initializeApp(firebaseConfig);
```

Comment:
prettier works? 
i see 4 tabs instead 2

Reply:
I checked - it works. in my file on my computer, 2 indents.
```const firebaseConfig = {
	apiKey: "YOUR_API_KEY",
	authDomain: "YOUR_AUTH_DOMAIN",
	projectId: "YOUR_PROJECT_ID",
	storageBucket: "YOUR_STORAGE_BUCKET",
	messagingSenderId: "YOUR_MESSAGING_SENDER_ID",
	appId: "YOUR_APP_ID",
};

firebase.initializeApp(firebaseConfig);```

Reply:
I checked - now it works. The Format On Save checkbox was unchecked.